content/docs/extensions/timescaledb.md

```diff
@@ -0,0 +1,235 @@
+---
+title: The timescaledb extension
+subtitle: timescaledb is an extension for handling time-series data.
+enableTableOfContents: true
+---
+
+`timescaledb` enables efficient storage and retrieval of time-series data. It is designed to handle large volumes of time-stamped data and provides SQL capabilities on top of a time-oriented data model such as IoT data, sensor readings, financial market data, and other time-series datasets.
+
+<Admonition type="note">
+`timescaledb` is an open-source extension for Postgres that can be installed on any Neon Project using the instructions below.
+</Admonition>
+
+**Version Availability:**
+
+The version of `timescaledb` available on Neon varies based on which version of Postgres you select for your project.
+
+- Postgres 14 - `timescaledb` 2.10.1
+- Postgres 15 - `timescaledb` 2.10.1
+- Postgres 16 - `timescaledb` 2.13.0
+
+_Only Apache-2 licensed features are supported. Compression is not supported._
+
+## Enable the `timescaledb` extension
+
+You can enable the extension by running the following `CREATE EXTENSION` statement in the Neon **SQL Editor** or from a client such as `psql` that is connected to Neon.
+
+```sql
+CREATE EXTENSION IF NOT EXISTS timescaledb;
+```
+
+For information about using the Neon SQL Editor, see [Query with Neon's SQL Editor](/docs/get-started-with-neon/query-with-neon-sql-editor). For information about using the `psql` client with Neon, see [Connect with psql](/docs/connect/query-with-psql-editor).
+
+## Basic functions for working with Hypertables and chunks
+
+Create a `hypertable` for temperature data:
+
+```sql
+CREATE TABLE temperature_data (
+ time       TIMESTAMPTZ NOT NULL,
+ location   TEXT        NOT NULL,
+ temperature DOUBLE PRECISION
+);
+```
+
+Create a `hypertable` for temperature data using `create_hypertable`:
+
+
+```sql
+SELECT create_hypertable('temperature_data', 'time');
+```
+
+Insert data in `temperature_data` table:
```

Comment:
Suggest looking at how the DO article provides context for inserting data and following that example.
https://www.digitalocean.com/community/tutorials/how-to-install-and-use-timescaledb-on-centos-7#step-4-writing-and-deleting-data
- Add similar context 
- Show how to insert a row using SQL
- Show how to import multiple rows
- Discusses / shows how to insert a large scale data inserts using a TimescaleDB-created dataset using psql \copy